src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -2010,8 +2037,8 @@ public void onAudioFocusChange(int type)
 		Log.d("VanillaMusic", "audio focus change: " + type);
 		switch (type) {
 		case AudioManager.AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK:
-			mDuckedLoss = (mState & FLAG_PLAYING) != 0;
-			unsetFlag(FLAG_PLAYING);
+			mDuckedLoss = true;
```

Comment:
Hmm.. i'd have to ask Chris ;-)
Most of them look pretty harmless as many of them only do UI Updates & Broadcasts.

But maybe it would be a good idea to get rid of mState anyway and replace it with a class/object